src/OpenTelemetry.Exporter.Zipkin/ZipkinExporterOptions.cs

```diff
@@ -30,11 +31,27 @@ public sealed class ZipkinExporterOptions
         internal const int DefaultMaxPayloadSizeInBytes = 4096;
 #endif
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ZipkinExporterOptions"/> class.
+        /// Initializes zipkin endpoint.
+        /// </summary>
+        public ZipkinExporterOptions()
+        {
+            try
+            {
+                this.Endpoint = new Uri(Environment.GetEnvironmentVariable("OTEL_EXPORTER_ZIPKIN_ENDPOINT") ?? "http://localhost:9411/api/v2/spans");
```

Comment:
@vishweshbankwar Would you mind moving `OTEL_EXPORTER_ZIPKIN_ENDPOINT` to a constant and then referencing in the unit tests?

https://github.com/open-telemetry/opentelemetry-dotnet/pull/2123 is doing the same.